app/bundles/CoreBundle/Templating/Helper/FormatterHelper.php

```diff
@@ -189,7 +189,7 @@ public function normalizeStringValue($string)
             return $this->_($stringIsDate, 'datetime');
         }
 
-        return $this->_($string);
```

Comment:
@kuzmany 
Using `_` function we are filtering and sanitising  html data and now we are using string without sanitisation.
is it ok to use html with sanitisation ?